### PR TITLE
Test infra - kube rules

### PR DESCRIPTION
**Pull request by oren-zohar**
Adds support for Kube-API rules tests to our automation infra

- fixes: https://github.com/elastic/cloudbeat/issues/154

### CODE REVIEW

File: tests/product/tests/conftest.py [hunk 1]
```diff
@@ -1,22 +1,22 @@
 import pytest
-import time
-from commonlib.io_utils import  get_k8s_yaml_objects
+from commonlib.io_utils import *
```
Comment (gurevichdmitry) [inline tests/product/tests/conftest.py:2]: @oren-zohar, according to PEP8 - imports section, wildcard imports should be avoided.
[PEP8 Imports](https://peps.python.org/pep-0008/#imports)

File: tests/product/tests/kube_rules.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+from dataclasses import dataclass, astuple
```
Comment (gurevichdmitry) [inline tests/product/tests/kube_rules.py:1]: @oren-zohar, In order to separate data and tests, may be a good idea to create data folder under product/tests and to move kube_rules.py file to this folder.

**Review by gurevichdmitry — COMMENTED** [review]
@oren-zohar , review is done

**Comment by oren-zohar** [comment]
@gurevichdmitry addressed your comments, Apropo PEP8, I ran pylint locally and addressed some violations, if you want we can add a CI step for it.

**Comment by gurevichdmitry** [comment]
@oren-zohar Using pylint is a good idea, but it is very strict and may fail CI. So we need to discuss it.

**Review by gurevichdmitry — APPROVED** [review]
LGTM

**Pull request merged**